modules/search/searchWorker.py

```diff
@@ -0,0 +1,16 @@
+from modules.search.Search import Search
+
+
+def searchWorker(pause, exitRequest, pipelineIn, pipelineOut):
+  
+    print("Start Search")
+    
+    search = Search()
+    
+    pause.acquire()
+    
+    values = pipelineIn.get()
```

Comment:
Can you put a comment here that outlines what values is going to look like. Just put the following shape:
```
{
  tentGPS: value,
  planeGPS: value,
  angle: value
}
```